### ISSUE TITLE
version 5.0.34 fails to execute when old change entries without executionMillis field are present in the database

### ISSUE THREAD

**Issue by pvanhees** [mongock/mongock#490]
## Description

When upgrading from mongock version 4.3.8 to 5.0.34 we get the following error when starting the application: 
```
at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:758)\n\t... 13 common frames omitted\nCaused by: java.lang.NullPointerException: Cannot invoke \"java.lang.Long.longValue()\" because the return value of \"org.bson.Document.getLong(Object)\" is null
at io.mongock.driver.mongodb.sync.v4.repository.MongoSync4ChangeEntryRepository.lambda$getEntriesLog$0(MongoSync4ChangeEntryRepository.java:117)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
at io.mongock.driver.mongodb.sync.v4.repository.MongoSync4ChangeEntryRepository.getEntriesLog(MongoSync4ChangeEntryRepository.java:120)
at io.mongock.driver.api.entry.ChangeEntryService.getEntriesMap(ChangeEntryService.java:67)
at io.mongock.driver.api.entry.ChangeEntryService.getExecuted(ChangeEntryService.java:37)
at io.mongock.runner.core.executor.operation.change.MigrationExecutorBase.loadExecutedChangeEntries(MigrationExecutorBase.java:424)
at io.mongock.runner.core.executor.operation.change.MigrationExecutorBase.executeMigration(MigrationExecutorBase.java:92)
at io.mongock.runner.core.executor.operation.change.MigrationExecutorBase.executeMigration(MigrationExecutorBase.java:47)
at io.mongock.runner.core.executor.MongockRunnerImpl.execute(MongockRunnerImpl.java:53)\n
```

I tried to  investigate the reason of the error and found out that it's probably because at some point in the `MongoSync4ChangeEntryRepository` the method `getEntriesLog` was introduced where changeEntries were loaded from the database. When there is a changeEntry in the DB that has no `executionMillis` field, the Spring Boot application startup will fail with the mentioned error because the getEntriesLog tries to convert a null value to a primitive long. 

The changeEntries in the database in our application with no `executionMillis` are quite old (2018) from the time where the `ChangeEntry` class didn't contain the `executionMillis` field yet. 

So the`MongoSync4ChangeEntryRepository` wrongly assumes that the `executionMillis` field will always be present in the DB.
## PRIORITY

NORMAL

### Mongock
- version: 5.0.34
- Modules involved: springboot, MongoDB
- annotation approach

## Steps to Reproduce

1. Insert a change entry in the database without executionMillis field
2. run the mongock migration

## Behaviour

**Expected behavior:** mongock is executed without errors when having a changelog with entries without executionMillis field

**Actual behavior:** mongock execution fails

**How often the bug happens:** reproducable every time

## Additional context

[commit where bug was introduced](https://github.com/mongock/mongock/commit/7cf557d87c26449cbfdc096ceb0ba354c0c4aa23#diff-8ef27daf1692cac49adff07d77d67c8b0105b7ff87fcb99972e86356de5704ea)

**Comment by dieppa** [comment]
Sorry, I closed this by mistake.

Please can you try the recently released version 5.0.35?

Thanks

**Comment by dieppa** [comment]
Hello @pvanhees , we assume this has been fixed as we haven't got any response 😄
